lib/topology_lib_ip/library.py

```diff
@@ -194,11 +318,40 @@ def add_link_type_vlan(enode, portlbl, name, vlan_id, shell=None):
     response = enode(cmd, shell=shell)
     assert not response
 
+from pytest import set_trace
+
+def show_interface(enode, dev):
+    """
+    Show a interface.
+
+    :param enode: Engine node to communicate with.
+    :type enode: topology.platforms.base.BaseNode
+    :param str dev: Unix network device name. Ex 1, 2, 3..
+    :return: A combined dictionary as returned by both
+     :func:`topology_lib_ip.parser.parse_ip_addr_show`
+     :func:`topology_lib_ip.parser.parse_ip_stats_link_show`
+    """
+
+    if dev is not None:
+        cmd = 'ip addr list dev {ldev}'.format(ldev=dev)
+        response = enode(cmd, shell='bash')
+    
+    first_half_dict = parse_ip_addr_show(response)
```

Comment:
Although there are no other callers yet, these parse functions should be useful tools to an expanding ip library. I'll mark them as private.